data-access/src/app/domain/contexts/cases/service-ticket/v1/service-ticket-v1.ts

```diff
@@ -106,7 +112,8 @@ export class ServiceTicketV1<props extends ServiceTicketV1Props> extends Aggrega
     community: CommunityEntityReference,
     property: PropertyEntityReference,
     requestor: MemberEntityReference,
-    context: DomainExecutionContext
+    context: DomainExecutionContext,
+    assignedVendor?: VendorUserEntityReference
```

Comment:
Remove this, we aren't setting assigned vendor on service ticket creation. Remove the logic for setting assignedVendor inside this function as well